docs/topics/wildfly-swarm/docs/howto/writing-an-application-from-scratch/src/main/resources/pom-product.xml

```diff
@@ -16,6 +16,7 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <failOnMissingWebXml>false</failOnMissingWebXml>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <fabric8.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:1.2</fabric8.generator.from>
```

Comment:
Needs to be changed upstream (in WildFly Swarm) and then synced.

Reply:
created jira for @tradej https://issues.jboss.org/browse/RHOARDOC-1277